src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -45,7 +45,7 @@ private Optional<EppoValue> getAssignmentValue(
             String subjectKey,
             String flagKey,
             EppoAttributes subjectAttributes,
-            Map<String, EppoAttributes> assignmentOptions
+            Map<String, EppoAttributes> actionsWithAttributes
```

Comment:
after coming back to it, I decided to rename this variable to be more bandit-centric and clear on what it is.